query/src/datasources/store_client/remote_factory.rs

```diff
@@ -20,8 +20,8 @@ use common_exception::Result;
 use common_flights::StoreClient;
 
 use crate::configs::Config;
-use crate::datasources::database::remote::StoreClientProvider;
-use crate::datasources::database::remote::TryGetStoreClient;
+use crate::datasources::store_client::StoreClientProvider;
+use crate::datasources::store_client::TryGetStoreClient;
```

Comment:
I knew the following `ClientProvider` part not changed in this patch, but it seems that it is not related to the remote_factory? Is it possible to move out from this file to a new or others file?

Reply:
shall I move the whole `query/src/datasources/store_client` dir to `common/store_api/`, and rename store_client to `store_client_helper` or `util`?

Reply:
or move it to `common/flights`?

Reply:
I am not sure which one is more make sense, decide by you :)

Reply:
Ok

Reply:
`ClientProvider` is split to `common/store-api` and  `common-flights`